src/fontra/views/editor/editor.html

```diff
@@ -268,6 +268,10 @@
       padding-right: 0.8em;
     }
 
+    .ui-list {
```

Comment:
If we change it to `sources-list`, it won't affect the glyph search results.
```suggestion
    .sources-list {
```
I'm not convinced we should set this globally for ui-list.